lc2fen/infer_pieces.py

```diff
@@ -456,11 +650,17 @@ def inferred_pieces_from_move(initial_sq, final_sq, action):
     initial_sq = (initial_sq // 8, initial_sq % 8)  # (row, column)
     final_sq = (final_sq // 8, final_sq % 8)
 
-    capturing = action.endswith("captures")
+    capturing = action.endswith("captures") | action.endswith("en_passants")
     white = action.startswith("white")
+    castling = "castles" in action
```

Comment:
There will always be 6 characters before the "castles" keyword, so I believe this can be slightly optimized to `action[6:13] == "castles"`

Reply:
Done!